zerver/lib/url_preview/oembed/__init__.py

```diff
@@ -11,4 +11,10 @@ def get_oembed_data(url: str,
         return None
 
     data['image'] = data.get('thumbnail_url')
+    type_ = data.get('type', '')
```

Comment:
I would call this `open_graph_type`?

Reply:
Yeah, It's definitely a better name.  I'll send in a refactor commit when I'm fixing up the vimeo previews stuff. Thanks!